bower.json

```diff
@@ -17,13 +17,15 @@
     "tests"
   ],
   "dependencies": {
-    "angular": "^1.5.3",
-    "angular-animate": "^1.5.3",
+    "angular": "1.5.5",
```

Comment:
I'd prefer we kept angular at "^1.5.5" .. and updated all the core angular bits to the same. Any reason for forcing =1.5.5?